openretina/models/core_readout.py

```diff
@@ -181,6 +183,7 @@ def __init__(
         dropout_rate: float = 0.0,
         maxpool_every_n_layers: Optional[int] = None,
         downsample_input_kernel_size: Optional[tuple[int, int, int]] = None,
+        data_info: dict[str, int] | None = None,
```

Comment:
```suggestion
        data_info: dict[str, Any] | None = None,
```

I think the int is wrong in the type annotation